src/tir/transforms/primfunc_utils.cc

```diff
@@ -30,6 +30,10 @@ namespace tir {
 namespace transform {
 transform::Pass BindTarget(Target target) {
   auto fpass = [target](tir::PrimFunc f, IRModule m, transform::PassContext ctx) {
+    if (f->GetAttr<Integer>(tvm::tir::attr::kIsHostFunc) == 1) {
+      return WithAttr(std::move(WithoutAttr(std::move(f), tvm::tir::attr::kIsHostFunc)),
+                      tvm::attr::kTarget, target->host.value_or(Target("llvm")));
```

Comment:
Is there a case where the target host is `None`?

Reply:
Yes, when we use target tags like `nvidia/geforce-rtx-3070` the default target host is `None`.

Reply:
got it. that makes sense!